src/main/kotlin/racinggame/application/RacingService.kt

```diff
@@ -9,25 +9,26 @@ object RacingService {
     private const val ERROR_CAR_COUNT = "차량 개수가 잘못되었습니다."
     private const val ERROR_ROUND_COUNT = "라운드 횟수가 잘못되었습니다."
 
-    var cars: MutableList<Car> = mutableListOf()
-    var rounds: MutableList<Round> = mutableListOf()
-
     fun start(
-        carCount: Int,
+        names: List<String>,
         moveCondition: MoveCondition,
         roundCount: Int,
     ) {
-        require(carCount > 0) { ERROR_CAR_COUNT }
+        require(!names.isEmpty()) { ERROR_CAR_COUNT }
```

Comment:
```suggestion
require(names.isNotEmpty()) { ERROR_CAR_COUNT }
```
kotlin에서는 isNotEmpty()를 이용해서 느낌표를 없애줄수도 있습니다. ㅎㅎ